### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by michalskrivanek**

### CODE REVIEW

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

File: README.md [hunk 1]
```diff
@@ -32,43 +39,17 @@ The release shall include all features of the Basic User Portal and selected fun
...
-    - deploy User Portal as a self-standing app
-    - integrate in ManageIQ
-    - customize for your own oVirt UI 
-- In comparison to recent GWT UserPortal:
-    - improved usability and Look&Feel
+- The project focuses on reusability, extensibility and simplicity. Compared to the current GWT User Portal it should provide:
+    - improved usability and look&feel
     - simplified maintenance & ongoing development
     - faster UI responses (incl. start-up)      
+    - easy customiza for your own oVirt UI implementation
```
Comment (matobet) [inline README.md]: *customization

**Comment by mareklibra** [comment]
Replaced by https://github.com/oVirt/ovirt-web-ui/pull/151

**Pull request closed without merging**